README.md

```diff
@@ -122,7 +142,7 @@ can act on it.
 ```lua
 local Worktree = require("git-worktree")
 
--- op = "switch", "create", "delete"
+-- op = Operation.Switch, Operation.Create, Operation.Delete
```

Comment:
should the switch and create have their arguments updated?  

op, path, upstream seems like it is not quite right for create anymore.

Reply:
to fix #24 i think we will need to add previous worktree and new worktree to the switch operation

We could look at updating others